src/services/services.ts

```diff
@@ -1967,6 +1967,77 @@ namespace ts {
         }
     }
 
+    function isObjectLiteralElement(node: Node): node is ObjectLiteralElement  {
+        switch (node.kind) {
+            case SyntaxKind.JsxAttribute:
+            case SyntaxKind.JsxSpreadAttribute:
+            case SyntaxKind.PropertyAssignment:
+            case SyntaxKind.ShorthandPropertyAssignment:
+            case SyntaxKind.MethodDeclaration:
+            case SyntaxKind.GetAccessor:
+            case SyntaxKind.SetAccessor:
+                return true;
+        }
+        return false;
+    }
+
+    function getNameFromObjectLiteralElement(node: ObjectLiteralElement) {

```

Comment:
Well, `getTextOfPropertyName(node.name) === `getNameFromObjectLiteralElement(node)`

---
In reply to: [88698797](https://github.com/Microsoft/TypeScript/pull/12107#discussion_r88698797) [](ancestors = 88698797)